Pipeline/CreateUCDComponentVersion/dbb-ucd-packaging.groovy

```diff
@@ -360,7 +405,93 @@ xml.manifest(type:"MANIFEST_SHIPLIST"){
+                                if (properties.buildReportOrder.size() != 1) {
+    
+                                    // Url to DBB Build result
+                                    if (buildResult != null) {
+                                        property(name : "dbb-buildResultUrl", label: buildResult.getLabel(), value : buildResult.getUrl())
+                                    }   
+                                    // Populate build result properties
+                                    if (buildResultProperties != null) {
+                                        buildResultProperties.each{
+                                            //not all properties need to be included in the shiplist
+                                            //can ignore files processed
+                                            //can ignore full build / impact build
+                                            property(name:it.key, value:it.value)
+                                        }
+                                    }
+                                }
+    
+                                property(name:"buildcommand", value:record.getAttribute("command"))
+                                property(name:"label", value:record.getAttribute("label"))
+    
+                                // add githash to container
+                                def githash = "" // set empty
+                                if (buildResultProperties != null){
+                                    // get git references from build properties
+                                    def gitproperty = buildResultProperties.find{
+                                        it.key.contains(":githash:") && record.getAttribute("file").contains(it.key.substring(9))
+                                    }
+                                    if (gitproperty != null ) {
+                                        githash = gitproperty.getValue()
+                                        // set properties in shiplist
+                                        property(name:"githash", value:githash)
+                                        if(properties.git_commitURL_prefix) property(name:"git-link-to-commit", value:"${properties.git_commitURL_prefix}/${githash}")
+                                    }
+                                }
+                                
+                                // add source information in the input column of UCD
+                                inputUrl = (buildResultProperties != null && properties.git_treeURL_prefix && githash!="") ? "${properties.git_treeURL_prefix}/${githash}/"+ record.getAttribute("file") : ""
+                                inputs(url : "${inputUrl}"){
+                                    input(name : record.getAttribute("file"), compileType : "Main", url : inputUrl)
+                                    
+                                    // adding dependencies
```

Comment:
I don't think that we capture a dependency sets for the ZFS files, do we?

Reply:
Right, removed.